exec/java-exec/src/main/java/org/apache/drill/exec/server/options/TypeValidators.java

```diff
@@ -90,6 +91,62 @@ public void validate(OptionValue v) {
     }
   }
 
+  public static class MinRangeDoubleValidator extends RangeDoubleValidator {
+    private final double min;
+    private final double max;
+    private final String maxValidatorName;
+
+    public MinRangeDoubleValidator(String name, double min, double max, double def, String maxValidatorName) {
+      super(name, min, max, def);
+      this.min = min;
+      this.max = max;
+      this.maxValidatorName = maxValidatorName;
+    }
+
+    @Override
+    public void validate(OptionValue v, final OptionManager manager) {
+      super.validate(v, manager);
+      if (manager != null) {
```

Comment:
In that case, the caller should not be using this validator because this would be the same `RangeDoubleValidator`.